EventFilter/GEMRawToDigi/interface/GEMRawToDigi.h

```diff
@@ -9,7 +9,11 @@
 class GEMRawToDigi {
 public:
   std::unique_ptr<gem::AMC13Event> convertWordToAMC13Event(const uint64_t* word);
+  bool vfatError() const { return vfatError_; }
+  bool amcError() const { return amcError_; }
```

Comment:
I do not see use cases.
Are these added for some future use or was some code accidentally not committed?

Reply:
It was used, but I removed the use cases when I reduced the LogWarnings. 
I would like to add a few of these checks to the onlineDQM.